### DISCUSSION TITLE
Welcome to crafting-dead Discussions!

### DISCUSSION THREAD

**Discussion by 0xE69** [nexusnode/crafting-dead#222] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maxvilt** [comment]
Hello, I'm using minecraft 1.16.5 and I really like your mod, but effects like broken bones and bleeding cause significant discomfort and errors. How can i turn off this effects? Answer me please.

**Reply by josephtarbit** [reply]
Take a look in the config (inside the configs folder in your world folder)
